.vscode/launch.json

```diff
@@ -7,11 +7,11 @@
         "name": "Azure CLI Debug (Integrated Console)",
         "type": "python",
         "request": "launch",
-        "pythonPath": "${config:python.pythonPath}",
+        "pythonPath": "${config:python.interpreterPath}",
```

Comment:
the pythonPath not work?

Reply:
It is automatically changed when I run the code, maybe because the code is ran in terminal of vscode?